testgrid/deploy/secrets.enc.yaml

```diff
@@ -0,0 +1,17 @@
+dockerhub_username: ENC[AES256_GCM,data:3Sam1XuphbK5GVEMCg==,iv:Ll9ELYnuJJfpQ2jeuW6N22FM1nRpqymKia2Zr1QaEIk=,tag:sYjTvtHep63wp0lKnhUv/w==,type:str]
+dockerhub_password: ENC[AES256_GCM,data:5i7eWfYOgBk6Kx5MuFvcmg30LTg=,iv:i96u9zBtCO+zZtXEvkNh4p0SwZ+Le3kViUT3Rzpa+7A=,tag:d4nluRJvZQ3KOG9Lg6ZvpQ==,type:str]
```

Comment:
Assuming this is encrypted and the key is not in this PR?

Reply:
Yes. With [sops](https://github.com/mozilla/sops)